Build error: undefined: prometheus.Handler

**Issue type**:
Bug report

**What happened**:
I got the following error while trying to build from source.
```
prometheus-nginxlog-exporter/main.go:189:26: undefined: prometheus.Handler
```

I'm building upstream.

```
$ go version
go version go1.13.6 linux/amd64

$ mkdir /tmp/gobuild

$ GOPATH=/tmp/gobuild go get github.com/martin-helmich/prometheus-nginxlog-exporter
# github.com/martin-helmich/prometheus-nginxlog-exporter
/tmp/gobuild/src/github.com/martin-helmich/prometheus-nginxlog-exporter/main.go:189:26: undefined: prometheus.Handler
```

--- a/go.mod
+++ b/go.mod
@@ -1,30 +1,41 @@
 module github.com/martin-helmich/prometheus-nginxlog-exporter
 
 require (
+	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d // indirect
 	github.com/armon/go-radix v1.0.0 // indirect
-	github.com/beorn7/perks v0.0.0-20160229213445-3ac7bf7a47d1 // indirect
+	github.com/creack/pty v1.1.9 // indirect
 	github.com/fsnotify/fsnotify v1.4.7 // indirect
-	github.com/golang/protobuf v0.0.0-20160614223140-0c1f6d65b5a1 // indirect
+	github.com/go-logfmt/logfmt v0.5.0 // indirect
+	github.com/golang/protobuf v1.3.3 // indirect
+	github.com/google/gofuzz v1.1.0 // indirect
 	github.com/hashicorp/consul v0.0.0-20150921174127-de080672fee9
 	github.com/hashicorp/go-msgpack v0.5.3 // indirect
 	github.com/hashicorp/golang-lru v0.5.1 // indirect
 	github.com/hashicorp/hcl v1.0.0
 	github.com/hpcloud/tail v1.0.0
-	github.com/kr/pretty v0.1.0 // indirect
-	github.com/matttproud/golang_protobuf_extensions v1.0.1 // indirect
-	github.com/prometheus/client_golang v0.0.0-20160916180340-5636dc67ae77
-	github.com/prometheus/client_model v0.0.0-20150212101744-fa8ad6fec335 // indirect
-	github.com/prometheus/common v0.0.0-20160623151427-4402f4e5ea79 // indirect
-	github.com/prometheus/procfs v0.0.0-20160411190841-abf152e5f3e9 // indirect
+	github.com/julienschmidt/httprouter v1.3.0 // indirect
+	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
+	github.com/kr/pretty v0.2.0 // indirect
+	github.com/kr/pty v1.1.8 // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v1.0.1 // indirect
+	github.com/mwitkow/go-conntrack v0.0.0-20190716064945-2f068394615f // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/prometheus/client_golang v1.4.0
 	github.com/satyrius/gonx v1.3.1-0.20180709120835-47c52b995fe5
 	github.com/smartystreets/goconvey v0.0.0-20190306220146-200a235640ff // indirect
+	github.com/stretchr/objx v0.2.0 // indirect
 	github.com/stretchr/testify v1.4.0
-	golang.org/x/sys v0.0.0-20161023150541-c200b10b5d5e // indirect
-	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
+	golang.org/x/crypto v0.0.0-20200204104054-c9f3fb736b72 // indirect
+	golang.org/x/mod v0.2.0 // indirect
+	golang.org/x/net v0.0.0-20200202094626-16171245cfb2 // indirect
+	golang.org/x/sys v0.0.0-20200202164722-d101bd2416d5 // indirect
+	golang.org/x/text v0.3.2 // indirect
+	golang.org/x/tools v0.0.0-20200205141839-4abfd4a1628e // indirect
 	gopkg.in/fsnotify.v1 v1.4.7 // indirect
 	gopkg.in/mcuadros/go-syslog.v2 v2.3.0
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
-	gopkg.in/yaml.v2 v2.2.7
+	gopkg.in/yaml.v2 v2.2.8
 )
 
 go 1.13

--- a/main.go
+++ b/main.go
@@ -34,6 +34,7 @@ import (
 	"github.com/martin-helmich/prometheus-nginxlog-exporter/relabeling"
 	"github.com/martin-helmich/prometheus-nginxlog-exporter/tail"
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/satyrius/gonx"
 )
 
@@ -76,9 +77,10 @@ func (m *Metrics) Init(cfg *config.NamespaceConfig) {
 	}, labels)
 
 	m.upstreamSeconds = prometheus.NewSummaryVec(prometheus.SummaryOpts{
-		Namespace: cfg.Name,
-		Name:      "http_upstream_time_seconds",
-		Help:      "Time needed by upstream servers to handle requests",
+		Namespace:  cfg.Name,
+		Name:       "http_upstream_time_seconds",
+		Help:       "Time needed by upstream servers to handle requests",
+		Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
 	}, labels)
 
 	m.upstreamSecondsHist = prometheus.NewHistogramVec(prometheus.HistogramOpts{
@@ -186,7 +188,7 @@ func main() {
 	listenAddr := fmt.Sprintf("%s:%d", cfg.Listen.Address, cfg.Listen.Port)
 	fmt.Printf("running HTTP server on address %s\n", listenAddr)
 
-	http.Handle("/metrics", prometheus.Handler())
+	http.Handle("/metrics", promhttp.Handler())
 
 	if err := http.ListenAndServe(listenAddr, nil); err != nil {
 		fmt.Printf("error while starting HTTP server: %s", err.Error())

